src/panels/config/scene/ha-config-scene.ts

```diff
@@ -38,9 +38,11 @@ class HaConfigScene extends HassRouterPage {
 
   private _computeScenes = memoizeOne((states: HassEntities) => {
     const scenes: SceneEntity[] = [];
-    Object.keys(states).forEach((entityId) => {
+    Object.entries(states).forEach(([entityId, state]) => {
```

Comment:
I don't think ES5 supports `Object.entries`. I suggest to use `Object.values` (since we polyfill that one in `compatibility.ts`) and get entityID from `state.entiy_id`